src/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -4,23 +4,89 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <ProjectGuid>{404DB891-B5AF-41E6-B89D-29E3F4573C4F}</ProjectGuid>
-    <!-- only supported by desktop CLR -->
-    <GeneratePlatformNotSupportedAssemblyMessage>SR.PlatformNotSupported_ReflectionContext</GeneratePlatformNotSupportedAssemblyMessage>
+    <!-- Only supported on .NET Framework and .NET Core -->
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetGroup)' == 'netstandard1.1' OR '$(TargetGroup)' == 'netstandard'">SR.PlatformNotSupported_ReflectionContext</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
Given these are just PNSE assemblies is there any reason to not just harvest them as well?

Reply:
The harvested ones don't throw with a message. I guess it is better to explain why it is throwing? If it doesn't matter I'll harvest them as well.

Reply:
I agree, let's build them for this package version to include the message and harvest them with the next version.